byterun/minor_gc.c

```diff
@@ -619,8 +619,10 @@ void caml_empty_minor_heap_domain (struct domain* domain)
           CAMLassert(Tag_val(vnew) == Infix_tag);
           v += offset;
         }
-        if (caml_atomic_cas_raw(*r,v,vnew)) ++rewrite_successes;
-        else ++rewrite_failures;
+        if (atomic_compare_exchange_strong((atomic_value*)*r, &v, vnew))
```

Comment:
Change it to `caml_atomic_cas` here to take advantage of `caml_domain_alone` info?